assets/styles/importer.less

```diff
@@ -184,6 +184,14 @@ td h3 {
   color: green;
 }
 
+[ng\:cloak], [ng-cloak], .ng-cloak {
```

Comment:
Hmmm, I'll merge this in and spend a bit of time trying ot make it look a bit nicer.